src/devices/video/ef9345.cpp

```diff
@@ -105,10 +107,25 @@ ef9345_device::ef9345_device(const machine_config &mconfig, const char *tag, dev
 	device_video_interface(mconfig, *this),
 	m_space_config("videoram", ENDIANNESS_LITTLE, 8, 16, 0, nullptr, *ADDRESS_MAP_NAME(ef9345)),
 	m_charset(*this, DEVICE_SELF),
+	m_variant(TYPE_EF9345),
 	m_palette(*this, finder_base::DUMMY_TAG)
 {
 }
 
+ef9345_device::ef9345_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock, uint32_t variant) :
```

Comment:
Please convert the other constructor to use C++11 constructor delegation (i.e. define it in terms of this constructor) to avoid the possibility member initialiser lists getting out of sync.